### ISSUE TITLE
What rule for check const variable was assigned?

### ISSUE THREAD

**Issue by hotoo** [eslint/eslint#3716]
When a `const` variable was assigned, give me a hint.

like:

```js
const C = 'CONST';

// ...

C = 'ANOTHER'; // hint me here.
```

I found the rule `prefer-const`, but I don't think if a variable not modified yet, it must a const variable.

And if have one of rule for this, I think it's would be default enabled.

**Comment by eslintbot** [comment]
Thanks for the issue! We get a lot of issues, so this message is automatically posted to each one to help you check that you've included all of the information we need to help you.

Reporting a bug? Please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
1. The source code that caused the problem
1. The configuration you're using (for the rule or your entire config file)
1. The actual ESLint output complete with line numbers

Requesting a new rule? Please be sure to include:

1. The use case for the rule - what is it trying to prevent or flag?
1. Whether the rule is trying to prevent an error or is purely stylistic
1. Why you believe this rule is generic enough to be included

Requesting a feature? Please be sure to include:

1. The problem you want to solve (don't mention the solution)
1. Your take on the correct solution to problem

Including this information in your issue helps us to triage it and get you a response as quickly as possible.

Thanks!

**Comment by mysticatea** [comment]
[no-const-assign](http://eslint.org/docs/rules/no-const-assign.html)

**Comment by hotoo** [comment]
Thank you :+1:
